opencanary/config.py

```diff
@@ -43,6 +44,11 @@ def detectIPTables():
         return False
 
 
+SERVICE_REGEXES = {
+    "ssh.version": r"SSH-(2.0|1.5|1.99|1.0)-([!-,\-./0-~]+(:?$|\s))(?:[ -~]*)$",
```

Comment:
Do we want to add the length constraint here too.
see https://github.com/thinkst/opencanary/pull/309/files#diff-5f3e79391c8c84f0500a2fbfaab887c2661e9e86cb9f26cdeb9cb6da69a0a308L194

Reply:
```
        if key == "ssh.version" and len(val) > 253:
            raise ConfigException(key, "SSH version string too long (%s..)" % val[:5])
```
We already have this which results in a nicer error message too. So going to leave it as is